plugins/techdocs-backend/src/service/DocsSynchronizer.ts

```diff
@@ -128,7 +128,13 @@ export class DocsSynchronizer {
         cache: this.cache,
       });
 
+      const interval = setInterval(() => {
```

Comment:
I liked the message's tone of voice 🙂 .
Would you mind adding a test case to [this](https://github.com/backstage/backstage/blob/6f097023fc9c5a1258f0697ea26435d468cb3a59/plugins/techdocs-backend/src/service/DocsSynchronizer.test.ts) file?

Reply:
done thank you :)